tests/Integration/ScriptsTest.php

```diff
@@ -291,7 +291,7 @@ public function test_wp_parsely_load_js_tracker_filter(): void {
 
 		// Since no secret is provided, the extra fields (inline scripts) on the
 		// loader should not be populated.
-		self::assertEquals( 1, count( $wp_scripts->registered['wp-parsely-loader']->extra ) );
+		self::assertEquals( 1, \is_array( $wp_scripts->registered['wp-parsely-loader']->extra ) || $wp_scripts->registered['wp-parsely-loader']->extra instanceof \Countable ? count( $wp_scripts->registered['wp-parsely-loader']->extra ) : 0 );
```

Comment:
I think we should create a function for it because we are using it in multiple places. I think SonarCloud will also complain about this 🤔